src/Microsoft.Azure.SignalR/ClientInvocation/CallerClientResultsManager.cs

```diff
@@ -174,6 +166,8 @@ public bool TryGetInvocationReturnType(string invocationId, out Type type)
             return false;
         }
 
+        public bool RemoveInvocation(string invocationId) => _pendingInvocations.TryRemove(invocationId, out _);
```

Comment:
when it returns `bool` use `TryRemoveInvocation`, or change `bool` to `void`

Reply:
done